corehq/apps/receiverwrapper/rate_limiter.py

```diff
@@ -101,7 +101,8 @@ def _delay_and_report_rate_limit_submission(domain, max_wait, datadog_metric):
         duration_tag = 'delayed_reject'
     datadog_counter(datadog_metric, tags=[
         'domain:{}'.format(domain),
-        'duration:{}'.format(duration_tag)
+        'duration:{}'.format(duration_tag),
```

Comment:
absolutely not blocking. but I recently learned about `f` strings from @millerdev 
`f'duration:{duration_tag}'` might be more readable? 😄

Reply:
Yeah they're new to me too! https://github.com/dimagi/commcare-hq/pull/26507